lte/gateway/c/oai/tasks/nas/emm/Attach.c

```diff
@@ -281,13 +279,18 @@ int emm_proc_attach_request(
     no_attach_proc.ue_id = ue_id;
     no_attach_proc.emm_cause = ue_ctx.emm_context.emm_cause;
     no_attach_proc.esm_msg_out = NULL;
-  OAILOG_ERROR(
-    LOG_NAS_EMM,
-    "EMM-PROC  - Sending Attach Reject to UE (ue_id = " MME_UE_S1AP_ID_FMT ")\n", ue_id);
+    OAILOG_ERROR(
```

Comment:
Not related to this diff, but I think we can update this to use `OAILOG_ERROR_UE` here. Since IMSI is available on this procedure we can use this to prepend the IMSI specified on the macro